core/unix/inc/TUnixSystem.h

```diff
@@ -31,11 +31,30 @@
 #include "TTimer.h"
 #endif
 
+#include <thread>
+#include <memory>
+
+int StackTraceExec(void *);
+
 typedef void (*SigHandler_t)(ESignals);
 
+struct TStackTraceHelper {
+   static const int             stringLength = 255;
+   char                         shellExec[stringLength];
```

Comment:
data member must start with a lower f and be camel case words, for example fStringLength and fShellExec.